bootstrap/lodesiempre.html

```diff
@@ -0,0 +1,306 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <link href="custom.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bitter:wght@700&family=Nunito:wght@400;600&display=swap" rel="stylesheet">
+
+    <title>ej2 bootstrap</title>
+  </head>
+  <body>
+
+    <header>
+      <div class="container">
+        <div class="row justify-content-center">
+          <div class="col-lg-4 col-sm-9">
+           <img src="onboarding-assets/img/titulo.png" width="90%" class="m-4 d-md-none">
```

Comment:
what about this?
https://getbootstrap.com/docs/5.0/content/images/

and some padding/margin if necessary